config/initializers/github_client.rb

```diff
@@ -0,0 +1,11 @@
+module Classroom
+  def self.github_client(options = {})
+    client_options = {
+      client_id:     Rails.application.secrets.github_client_id,
+      client_secret: Rails.application.secrets.github_client_secret,
+      auto_paginate: true
```

Comment:
Doesn't look like https://github.com/education/classroom/pull/521/files#diff-9955ca00c1bcd788fb13273ca8ace816L243 had `auto_paginate` enabled. Do you expect any issues with that?